.github/workflows/test.yml

```diff
@@ -53,6 +54,9 @@ jobs:
       - name: Install ansible and molecule
         run: pip3 install ansible molecule[docker] docker
 
+      - name: Install ansible-galaxy dependencies
+        run: ansible-galaxy collection install community.mongodb:1.3.0 --force
```

Comment:
Doesn't molecule handle this (`dependencies.name: galaxy` with `requirements.yml`)?

Reply:
I really hoped so but whenever I removed it, it failed due to a recent change in the mongodb collection. Unless you can see I'm doing something wrong else-where with Ansible requirements management.

Reply:
Maybe there's a bug somewhere 😢